bundle/python_bundle_workflow/scripts/inference.py

```diff
@@ -98,7 +98,10 @@ def _get_property(self, name, property):
                 value = getattr(self, f"get_{name}")()
             except AttributeError:
                 if property[BundleProperty.REQUIRED]:
-                    raise ValueError(f"unsupported property '{name}' is required in the bundle properties.")
+                    raise ValueError(
+                        f"unsupported property '{name}' is required in the bundle properties,"
```

Comment:
Nit: the word property we can replace with attribute as you removed properties ?

Reply:
Hi @vfdev-5 ,

I see your point.
But we are using `property` in the context:
https://github.com/Project-MONAI/MONAI/blob/dev/monai/bundle/properties.py#L30

Thanks.